DQMOffline/Trigger/plugins/TopMonitor.h

```diff
@@ -43,6 +43,10 @@
 #include "DataFormats/VertexReco/interface/VertexFwd.h"
 #include "DQMOffline/Trigger/plugins/TriggerDQMBase.h"
 
+//ATHER                                                                                                                                                                                                            
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DQMServices/Core/interface/DQMStore.h"
```

Comment:
why do you need it ?

Reply:
ValueMap is type of elecID. 

And I think it failed compiling without DQMStore for some reason but now it seems to be compile without this header file.

Reply:
please, delete this line